incubator/ethereum/values.yaml

```diff
@@ -0,0 +1,37 @@
+# Default values for ethereum.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+imagePullPolicy: IfNotPresent
+
+bootnode:
+  image:
+    repository: ethereum/client-go
+    tag: alltools-v1.7.3
+
+ethstats:
+  image:
+    repository: ethereumex/eth-stats-dashboard
+    tag: latest
```

Comment:
You'd want to pin to a release tag instead? Is `latest` a stable tag?

Reply:
Fair point. Let me reach out to the image owner to get them to push a versioned release.